lib/agents/DDQNagent.py

```diff
@@ -29,16 +40,48 @@ def __init__(self, state_size, action_size, mode):
 
     self.tensorboard = ModifiedTensorBoard(log_dir="logs/{}".format(time.time()))
   def remember(self, state, action, reward, next_state, done):
+    """
+    Appends metrics to memory.
+    
+    Args:
+        state (int): int representing state to act in.
+        action (int): int representing action taken.
+        reward (int): int representing reward after taking the action in the given state.
+        next_state (int): int representing the state reached after taking an action in the previous state.
+        done (bool): Whether or not agent has reached maximum number of steps.
+    
+    Returns:
+        None
+    """
     self.memory.append((state, action, reward, next_state, done))
   def act(self, state):
+    """
+    Takes an action, sometimes randomly.
+    
+    Args:
+        state (int): int representing state to act in.
+        
+    Returns:
+        Action taken, represented as an int.
+    """ 
     # Do something randomly
     if np.random.rand() <= self.epsilon and self.mode == "train":
       return random.randrange(self.action_size)
     act_values = self.online_net.predict(state)
     return np.argmax(act_values[0])  # returns action
     
   def replay(self, batch_size=32):
-    """ TODO vectorized implementation; 30x speed up compared with for loop """
+    """
+    Experience replay. Stores batch_size random samples from target network, freezes target network weights while training online network on the batch.
```

Comment:
Use flake8 as your linter in VSCODE.
To turn on: ctrl shift p -> select linter -> flake8
or just follow the instructions online.
The idea is every line should not be more than 80 characters (I think)